cli-artemis-jms/src/main/java/com/redhat/mqe/acc/AccSenderClient.java

```diff
@@ -99,7 +99,8 @@ public void startClient() {
                     msgProducer.send(message);
                 } catch (JMSException jex) {
                     LOG.error(jex.getCause().toString());
-                    if (jex.getCause() instanceof ActiveMQUnBlockedException) {
+                    if (jex.getCause() instanceof ActiveMQUnBlockedException &&
+                        ((ActiveMQUnBlockedException) jex.getCause()).getType().equals(ActiveMQExceptionType.UNBLOCKED)) {
```

Comment:
Why do we need `equals(ActiveMQExceptionType.UNBLOCKED)`? The instanceof is not enough?

Reply:
I think was with Core having some problem during failover (missing exactly 1 message).
Resend - for core protocol.